lua/cfc_random_spawn/sv_config.lua

```diff
@@ -22,42 +22,25 @@ CFCRandomSpawn.Config.CUSTOM_SPAWNS["gm_bluehills_test3"] = {
     { spawnPos = Vector( -1000.3621826172, 1044.5682373047,  464.03125 ) }
 }
 
-CFCRandomSpawn.Config.CUSTOM_SPAWNS["gm_novenka"] = {
-    { spawnPos = Vector( 1374, 7406, -384 ) }, -- Angle( 0, -180, 0 )
-    { spawnPos = Vector( 1680, 7765, -384 ) }, -- Angle( 0, 90, 0 )
-    { spawnPos = Vector( 1912, 8614, -384 ) }, -- Angle( 0, 180, 0 )
-    { spawnPos = Vector( 1923, 9188, -384 ) }, -- Angle( 0, -180, 0 )
-    { spawnPos = Vector( 1887, 10007, -384 ) }, -- Angle( 0, 180, 0 )
-    { spawnPos = Vector( 2269, 10641, -384 ) }, -- Angle( 0, -180, 0 )
-    { spawnPos = Vector( 2933, 10829, -384 ) }, -- Angle( 0, 135, 0 )
-    { spawnPos = Vector( 5383, 11632, -384 ) }, -- Angle( 0, -45, 0 )
-    { spawnPos = Vector( 6108, 11502, -384 ) }, -- Angle( 0, -135, 0 )
-    { spawnPos = Vector( 5479, 10526, -384 ) }, -- Angle( 0, 45, 0 )
-    { spawnPos = Vector( 5657, 8442, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( 5694, 7804, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( 6397, 7841, -384 ) }, -- Angle( 0, 180, 0 )
-    { spawnPos = Vector( 5629, 7036, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( 6415, 6208, -384 ) }, -- Angle( 0, -180, 0 )
-    { spawnPos = Vector( 4546, 6546, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( 4397, 6035, -384 ) }, -- Angle( 0, -45, 0 )
-    { spawnPos = Vector( 4186, 5866, -384 ) }, -- Angle( 0, -45, 0 )
-    { spawnPos = Vector( 4030, 5333, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( 3218, 4862, -384 ) }, -- Angle( 0, -90, 0 )
-    { spawnPos = Vector( 3465, 4310, -384 ) }, -- Angle( 0, 45, 0 )
-    { spawnPos = Vector( 2748, 4293, -384 ) }, -- Angle( 0, 135, 0 )
-    { spawnPos = Vector( 2398, 5662, -384 ) }, -- Angle( 0, -90, 0 )
-    { spawnPos = Vector( 2135, 5672, -384 ) }, -- Angle( 0, -45, 0 )
-    { spawnPos = Vector( 2651, 5660, -384 ) }, -- Angle( 0, -135, 0 )
-    { spawnPos = Vector( 935, 5040, -384 ) }, -- Angle( 0, -135, 0 )
-    { spawnPos = Vector( 696, 6400, -384 ) }, -- Angle( 0, 180, 0 )
-    { spawnPos = Vector( 54, 6402, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( -499, 7236, -384 ) }, -- Angle( 0, 90, 0 )
-    { spawnPos = Vector( -1042, 7049, -384 ) }, -- Angle( 0, 135, 0 )
-    { spawnPos = Vector( -2026, 7021, -384 ) }, -- Angle( 0, 45, 0 )
-    { spawnPos = Vector( -2446, 7283, -384 ) }, -- Angle( 0, 45, 0 )
-    { spawnPos = Vector( -2340, 8198, -384 ) }, -- Angle( 0, 0, 0 )
-    { spawnPos = Vector( -2435, 9145, -384 ) }, -- Angle( 0, 45, 0 )
-    { spawnPos = Vector( 3910, 7229, -184 ) }, -- Angle( 0, 135, 0 )
-    { spawnPos = Vector( 3872, 6256, -240 ) }, -- Angle( 0, 90, 0 )
-    { spawnPos = Vector( 2990, 8304, -264 ) } -- Angle( 0, 135, 0 )
+CFCRandomSpawn.Config.CUSTOM_SPAWNS["gm_novenka"] = { -- strawwagm_venka
+    { spawnPos = Vector( 6052, 11677, -383 ), spawnAngle = Angle( 0, 0, -135 ), pvp = true },
```

Comment:
Shouldn't the spawn angle be in the yaw and not the roll...? Unless we want people to spawn sideways I guess...

Reply:
@brandonsturgeon @StrawWagen